packages/html/src/StyledTable.ts

```diff
@@ -0,0 +1,53 @@
+import { SimpleTable } from "./SimpleTable";
+
+export class StyledTable extends SimpleTable {
+    constructor() {
+        super();
+    }
+    update(domNode, element) {
+        super.update(domNode, element);
+
+        this.theadColumnStyles().forEach((styleObj, i) => {
+            this.applyStyleObject(element.select(`.th-${i}`), styleObj);
+        });
+        this.tbodyColumnStyles().forEach((styleObj, i) => {
+            this.applyStyleObject(element.selectAll(`.col-${i}`), styleObj);
+        });
+        if (this.evenRowStyles_exists()) {
```

Comment:
Why the exists for this case (and not the others)?

Reply:
Question:  How to styles get "unset"?

Reply:
> Why the exists for this case (and not the others)?

The two exist conditionals are meant to avoid the expensive dom element selection, but they always evaluate to true because they're not optional. 

> Question: How to styles get "unset"?

Styles can't currently be unset. Though I'll add that feature when I amend this PR to remove the `_exist` conditions.